plugins/home/src/extensions.tsx

```diff
@@ -48,6 +48,21 @@ export type RendererProps = { title: string } & ComponentParts;
  */
 export type CardExtensionProps<T> = ComponentRenderer & { title?: string } & T;
 
+/**
+ * @public
+ */
+export type CardLayout = {
+  width?: { minColumns?: number; maxColumns?: number; defaultColumns?: number };
+  height?: { minRows?: number; maxRows?: number; defaultRows?: number };
+};
+
+/**
+ * @public
+ */
+export type CardSettings = {
+  schema?: any;
+};
```

Comment:
not blocking - this could be of type `RSJFTypeSchema` instead of a `any` here, right? If not is there any other more specific type than `any` possible?

Reply:
Yeah just had same comment from @freben. Might be something to improve in near future!

Reply:
Changed to RJSFSchema